messages/list.json

```diff
@@ -16,5 +16,7 @@
   "noActiveScratchOrgs": "No active scratch orgs found. Specify --all to see all scratch orgs",
   "deleteOrgs": "You have %s expired or deleted local scratch org authorizations. To remove authorizations for inactive orgs, run force:org:list --clean.",
   "noOrgsFound": "No orgs can be found.",
-  "noOrgsFoundAction": "Use one of the commands in force:auth or force:org:create to add or create new scratch orgs."
+  "noOrgsFoundAction": "Use one of the commands in force:auth or force:org:create to add or create new scratch orgs.",
```

Comment:
Suggested edit: "Use one of the auth commands or force:org:create to add or create a scratch org." 

(we shouldn't be mentioning "force:auth" anymore, right?  Also "create new" is redundant :) )